mail_tracking/static/src/xml/mail_tracking.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- © 2016 Antonio Espinosa - <antonio.espinosa@tecnativa.com>
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html). -->
+<template>
```

Comment:
You can see mail.xml https://github.com/odoo/odoo/blob/8.0/addons/mail/static/src/xml/mail.xml, for instance